.github/workflows/github-release.yml

```
@@ -1,50 +1,75 @@
-##
-## Create the github release and store artifact files (with checksum)
-##
+## Github workflow to create a github release and upload binary artifacts
 
 name: Github Release
 
-# Only run when:
-#   - manually triggered via the ci.yml workflow
-
 on:
   workflow_call:
     inputs:
       tag:
-        required: true
-        type: string
-      arch:
-        description: "Stringified JSON object listing of platform matrix"
+        description: "Release Tag"
         required: true
         type: string
     secrets:
       GH_TOKEN:
         required: true
 
+concurrency:
+  group: github-release-${{ github.head_ref || github.ref }}
```

Is this intended to be:
```suggestion
  group: github-release-${{ github.head_ref || github.ref || github.run_id}}
```

 following the concurrency pattern set in your other workflows?

---

i'll have to check - it may be a typo